tests/testthat/test-package.R

```diff
@@ -4,3 +4,9 @@ test_that("duplicate squares are removed", {
   x <- c("a", "a", "b")
   expect_identical(c("a", "b"), vet_squares(x))
 })
+
+
+test_that("each topic contains 24+ unique values", {
+  x <- letters[1:23]
+  expect_error(bingo(words = x), "m >= n_sq is not TRUE")
+})
```

Comment:
This is a good start on something that is clearly much needed.

I'd tackle this in a separate PR because it's bit of a job. Harder than editing the README. We need to assess the viability of everything below `inst/topics/` in a way that produces a decent error message, i.e. actually identifies the culprit(s), if something is wrong.